sdk/core/azure-core/inc/azure/core/http/websockets/websockets.hpp

```diff
@@ -311,13 +311,12 @@ namespace Azure { namespace Core { namespace Http { namespace WebSockets {
        */
       void Open(Azure::Core::Context const& context = Azure::Core::Context{});
 
-      /** @brief Closes a WebSocket connection to the remote server gracefully.
```

Comment:
Can `WebSocket` calss be made `final`?
Otherwise, please make the destructor `virtual` (or `protected`).

Reply:
Consider reviewing neighbor classes for the same thing.